src/components/AddFormRequest.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+
+class AddFormRequest extends React.Component {
+  render() {
+    return (
+      <div className='add-form'>
```

Comment:
Notice how many divs we have here for something that could be written <TextField value={this.state.title} onChange={this.handleChange} />